README.md

```diff
@@ -10,7 +10,7 @@ A simple dataflow engine with scalable semantics.
 The goal of pydra is to provide a lightweight Python dataflow engine for DAG construction, manipulation, and distributed execution.
 
 Feature list:
-1. Python 3.7+ using type annotation and dataclasses
+1. Python 3.7+ using type annotation and [attrs](https://www.attrs.org/en/stable/)
```

Comment:
Do we still use 3.7+ features, or can we relax to 3.6? (I know that at this point 3.7 isn't cutting edge, so maybe it doesn't matter.)

Reply:
my current intent was to get this merged, then do additional trimming/testing/relaxing. i did mention the 3.6 expansion to dorota when we met.